### ISSUE TITLE
Combine multiple multiqc reports?

### ISSUE THREAD

**Issue by astewart-twist** [ewels/MultiQC#723]
Hi @ewels! 

Is there a relatively straight forward way to combine a couple existing multiqc reports into a single report?  It's simple enough to just regenerate a single report by combining the multiple source datasets, but just curious if it could be merged from the multiqc reports alone.

**Comment by astewart-twist** [comment]
Well MegaQC sounds even more awesome than my original question!  I’ll have
to check it out.

Thanks!

**Comment by complexgenome** [comment]
@ewels  
Sorry to comment on a closed issue.
Are there any advances on this? I'm interested to create a cohort level multiqc report.

**Comment by ewels** [comment]
Yes - it's not complete yet but it's planned and some initial work has started.

See https://github.com/MultiQC/MultiQC/issues/1256 to track progress.
